test/unidirectional-streams.spec.js

```diff
@@ -93,16 +92,16 @@ describe('unidirectional streams', function () {
       await writer.write(buf)
     }
 
-    await new Promise((resolve) => setTimeout(resolve, 500))
+    await new Promise((resolve) => setTimeout(resolve, 2000))
```

Comment:
The longer timeout shouldn't be necessary as after this we wait for the server to close the connection after it's done reading, rather than when the client thinks it's done writing.